crates/cli/src/cfg.rs

```diff
@@ -115,6 +129,8 @@ impl Config {
     fn initialize(path: &str, network_name: &str) -> Self {
         Config {
             webserver: Some(WebServerConfig::default()),
+            keystore_type: Some("keychain".to_string()),
+            keystore_path: Some("~/.prism/keystore.json".to_string()),
```

Comment:
Question: The config now stores in subdirectories of ~/.prism (per network id), does this mean the default will be a keystore across all networks?

Reply:
I'm now using the path dependent on the network. See [ea38510](https://github.com/deltadevsde/prism/pull/216/commits/ea3851032bed3680b5c59458d492058e22c2b80e).